src/fsharp/FSharp.DependencyManager/FSharp.DependencyManager.Utilities.fs

```diff
@@ -230,6 +230,10 @@ namespace lib"
   <PropertyGroup>
     <TargetFramework>$(TARGETFRAMEWORK)</TargetFramework>
     <IsPackable>false</IsPackable>
+
+    <!-- Temporary fix some sdks, shipped internally with broken parameterization -->
+    <FSharpCoreImplicitPackageVersion Condition=""'$(FSharpCoreImplicitPackageVersion)' == '{{FSharpCoreShippedPackageVersion}}'"">4.7.0</FSharpCoreImplicitPackageVersion>
```

Comment:
This isn't going to do what you think.  At this point you're inside a string in a `.fs` file; you need to have already computed the variable expansion `{{FSharpCoreShippedPackageVersion}}`.